### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by jon4hz** [coinfoundry/miningcore#1275] [General]
Hi!

I've seen that there are a few useful bugfixes in the dev branch. Especially https://github.com/oliverw/miningcore/commit/14f4b26f0c375fe9a2bd8c41fc388f07681a7ab0. 
Would it be possible to merge dev to master and create a new release? 

Thanks a lot in advance!

( I know this isn't really a discussion but it isn't a bug report either, so...)

**Comment by oliverw** [comment]
Soon. Would like to include an Ergo fix.

**Reply by jon4hz** [reply]
Sure thing! Thanks for the reply :D
